cypress/fixtures/tmp/data

```diff
@@ -0,0 +1 @@
+/Users/maharielrosario/Development/vets-website/src/applications/disability-benefits/2346/tests/data
```

Comment:
This is using an exact path to something on your local machine

Reply:
Yeah I'm actually removing this file and the example upload from the PR. Shouldn't have carried over from my other branch.